src/test/java/seedu/address/storage/JsonAdaptedPersonTest.java

```diff
@@ -16,14 +16,15 @@
 import seedu.address.model.person.Email;
 import seedu.address.model.person.Name;
 import seedu.address.model.person.Phone;
+import seedu.address.model.person.TelegramHandle;
 
 public class JsonAdaptedPersonTest {
     private static final String INVALID_NAME = "R@chel";
     private static final String INVALID_PHONE = "+651234";
     private static final String INVALID_ADDRESS = " ";
     private static final String INVALID_EMAIL = "example.com";
-    private static final String INVALID_TELEGRAM = "rachel";
-    private static final String INVALID_TAG = "#friend";
+    private static final String INVALID_TELEGRAM_HANDLE = "rachel";
```

Comment:
Nice that you took care of the naming.

Reply:
It now feels quite long, so probably during the field overhaul by Sean, we could standardise it.

Reply:
Ok. I am going to approve this and await changes on his side then.